kong/clustering/services/sync/validate.lua

```diff
@@ -44,6 +44,16 @@ local function validate_deltas(deltas, is_full_sync)
     local delta_entity = delta.entity
 
     if delta_entity ~= nil and delta_entity ~= null then
+
+      -- validate workspace id
+      local ws_id = delta_entity.ws_id or delta.ws_id
+      if not ws_id or ws_id == null then
```

Comment:
should we check a special value ""?

Reply:
not need check "", it works well currently, extra task is here: https://konghq.atlassian.net/browse/KAG-6415

Reply:
Does Koko not generate `""` in this case?